controllers/argocd/deployment.go

```diff
@@ -1537,3 +1562,66 @@ func updateNodePlacement(existing *appsv1.Deployment, deploy *appsv1.Deployment,
 		*changed = true
 	}
 }
+
+func getRolloutInitContainer() []corev1.Container {
+	return []corev1.Container{
+		{
+			Name:  "rollout-extension",
+			Image: common.ArgoCDRolloutExtensionImage,
+			Env: []corev1.EnvVar{
+				{
+					Name:  "EXTENSION_URL",
+					Value: "https://github.com/argoproj-labs/rollout-extension/releases/download/v0.3.5/extension.tar",
```

Comment:
Can we move this string to `common/defaults.go`? Probably below `ArgoCDRolloutExtensionImage` var. It would be easy to update these values in future.